beacontools/scanner.py

```diff
@@ -100,6 +103,19 @@ def run(self):
             if event == LE_META_EVENT and subevent == EVT_LE_ADVERTISING_REPORT:
                 # we have an BLE advertisement
                 self.process_packet(pkt)
+        self.socket.close()
+
+    def set_scan_parameters(self, scan_type=ScanType.ACTIVE, interval_and_window_ms=(10, 10,),
+                            mac_type=BluetoothAddressType.RANDOM, filter_type=ScanFilter.ALL):
+        """"sets the scan parameters (socket, type, interval, window, own_type, filter)"""
+        from struct import pack
+        interval_ms, window_ms = interval_and_window_ms
+        scan_parameter_pkg = pack(
+            "<BBBBBBB",
+            scan_type, 0x0, interval_ms * MS_FRACTION_MULTIPLIER,
```

Comment:
shouldn't it be ```interval_ms / MS_FRACTION_MULTIPLIER```? since 10 / 0,625 = 16 = 0x10.
Also, this crashes since you need to round before passing the result to struct.pack():
```
Traceback (most recent call last):
  File "/usr/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/home/pi/dev/beacontools_pr/beacontools/scanner.py", line 96, in run
    self.set_scan_parameters()
  File "/home/pi/dev/beacontools_pr/beacontools/scanner.py", line 116, in set_scan_parameters
    0x0, window_ms * MS_FRACTION_MULTIPLIER, mac_type, filter_type)
struct.error: required argument is not an integer